rider/src/main/kotlin/com/jetbrains/rider/plugins/godot/FrontendBackendHost.kt

```diff
@@ -35,26 +43,42 @@ class FrontendBackendHost(project: Project) : ProtocolSubscribedProjectComponent
             }
         }
 
+        model.onTestRunnerOutputEvent.advise(projectComponentLifetime) {
+            myDebugProcessProperty.value?.console?.tryWriteMessageToConsoleView(
+                OutputMessageWithSubject(
+                    output = "${it.message}\r\n",
+                    type = when (it.type) {
+                        TestRunnerOutputEventType.Message -> OutputType.Info
+                        TestRunnerOutputEventType.Error -> OutputType.Error
+                    },
+                    subject = OutputSubject.Default
+                )
+            )
+        }
+
         model.startDebuggerServer.set { lt, _ ->
             val task = RdTask<Int>()
             val runManager = RunManager.getInstance(project)
             val configurationType = ConfigurationTypeUtil.findConfigurationType(MonoRemoteConfigType::class.java)
             val runConfiguration = runManager.createConfiguration(
                 GodotRunConfigurationGenerator.ATTACH_CONFIGURATION_NAME,
-                configurationType.factory
+                GodotDotNetRemoteConfigurationFactory(configurationType)
             )
-            val remoteConfiguration = runConfiguration.configuration as DotNetRemoteConfiguration
+            val remoteConfiguration = runConfiguration.configuration as GodotDotNetRemoteConfiguration
             remoteConfiguration.listenPortForConnections = true
             remoteConfiguration.port = NetUtils.findFreePort(500013)
             remoteConfiguration.address = "127.0.0.1"
 
             val processTracker: RiderDebugActiveDotNetSessionsTracker =
                 RiderDebugActiveDotNetSessionsTracker.getInstance(project)
-            processTracker.dotNetDebugProcesses.change.advise(projectComponentLifetime) { (event, debugProcess) ->
+            processTracker.dotNetDebugProcesses.change.advise(lt) { (event, debugProcess) ->
                 if (event == AddRemove.Add) {
                     debugProcess.initializeDebuggerTask.debuggerInitializingState.advise(lt) {
```

Comment:
Could you please subscribe on DotNetDebugProcess.sessionInfo instead? Not-null sessionInfo means that debugger is initialized (and you can get process id from it)